lib/elastic_apm/filters/secrets_filter.rb

```diff
@@ -19,8 +19,11 @@ class SecretsFilter
         /^\d{4}[- ]?\d{4}[- ]?\d{4}[- ]?\d{4}$/ # (probably) credit card number
       ].freeze
 
+      attr_reader :config
+
       def initialize(config)
         @config = config
+        @key_filters = KEY_FILTERS + config.custom_header_filters
```

Comment:
So it called `custom_header_filters`, but it worked for all type of keys. Thats not problem if you want to have separated configuration for headers, params, values in future.

Reply:
You are right. I'll consider moving back.